sample/EventHubTrigger-Batch/function.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bindings": [
+        {
+            "type": "eventHubTrigger",
+            "name": "input",
+            "cardinality": "many",
```

Comment:
By specifying this, the EventHub binding will use an array, so a batch of messages are passed to the function, rather than singletons.